src/RestWrite.js

```
@@ -540,13 +542,77 @@ RestWrite.prototype.ensureUniqueAuthDataId = async function () {
+      user.id = req.auth.user.id;
+      await user.fetch({ useMasterKey: true });
+      return user.get('authData') || {};
+    } catch (e) {
+      if (req.originalData && req.originalData.authData !== undefined) {
+        return req.originalData.authData || {};
+      }
+      return {};
+    }
+  } else if (req.originalData && req.originalData.authData !== undefined) {
+    return req.originalData.authData || {};
+  } else if (req.auth && req.auth.user) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)